### PR TITLE
Update feature config (compatible with Lhotse PR #525)

### PR DESCRIPTION
**Pull request by pzelasko**
Making things compatible with Lhotse PR https://github.com/lhotse-speech/lhotse/pull/525

Only yesno needed adaptation because it's 8kHz sampling rate, all the other recipes have 16kHz which is the default.

### CODE REVIEW

**Comment by pzelasko** [comment]
@csukuangfj @danpovey I'll let you merge this; note that it will require updating Lhotse version to latest master (I'm getting close to v1.0 release, so this is a good moment to make breaking changes like these).

File: egs/yesno/ASR/local/compute_fbank_yesno.py [hunk 1]
```diff
@@ -42,7 +42,9 @@ def compute_fbank_yesno():
     )
     assert manifests is not None
 
-    extractor = Fbank(FbankConfig(num_mel_bins=num_mel_bins))
+    extractor = Fbank(
+        FbankConfig(sampling_rate=8000, num_mel_bins=num_mel_bins)
```
Comment (csukuangfj) [inline egs/yesno/ASR/local/compute_fbank_yesno.py:46]: Is the purpose of specifying the sampling rate to pre-compute some constants during the constructor?

Please also update the fbank config used in the on the fly feature extractor in asr_datamodule.py in yesno.

Comment (pzelasko) [reply egs/yesno/ASR/local/compute_fbank_yesno.py:46]: Yeah, it precomputes the mel bands. I think the same thing is being done in kaldifeat.

Will update, thanks.

**Comment by csukuangfj** [comment]
I am merging it to avoid breaking the CI.

**Pull request merged**